client/src/components/BookCard.js

```diff
@@ -28,15 +36,17 @@ function BookCard({ cover, title, author, onCardBtnClick, buttonName }) {
         gridPosition="end"
         onClick={onCardBtnClick}
       ></RectangleButton>
+      <RectangleButton
+        buttonName={buttonMoreName}
```

Comment:
I don't understand what this button is for. Perhaps you could think of a more descriptive name. Is it a button, or a function?